### ISSUE TITLE
Problem with multi-word model names

### ISSUE THREAD

**Issue by webbtj** [laralib/l5scaffold#21]
Love the tool, HUGE time saver.

When generating a scaffold for a model with multiple words in the name (example "Email Message") the schema generation disagrees with Eloquent's standards for the table name. This examples creates a model called `EmailMessage`, and (without editing the migration) a table called `emailmessages`. Eloquent, by default expects the corresponding table to be snake case, so it expects `email_messages`.

**Comment by webbtj** [comment]
never mind, this was addressed with pull #5, which apparently I did back in May. The only actual problem is that there hasn't been a new release since then. So installing without specifying a version will of course use 1.0.2, do get this camel case fix you need to explicitly use version `dev-master`.

example:

```json
    "require-dev": {
        "fzaninotto/faker": "~1.4",
        "mockery/mockery": "0.9.*",
        "phpunit/phpunit": "~4.0",
        "phpspec/phpspec": "~2.1",
        "laralib/l5scaffold": "dev-master",
        "laracasts/generators": "^1.1"
    },
```

**Comment by fernandobritofl** [comment]
This was fixed in last version "dev-master"
tks
